packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -337,33 +337,6 @@ export class DeltaManager
         return this.connection.mode;
     }
 
-    /**
-     * Tells if container is in read-only mode.
-     * Data stores should listen for "readonly" notifications and disallow user
-     * making changes to data stores.
-     * Readonly state can be because of no storage write permission,
-     * or due to host forcing readonly mode for container.
-     * It is undefined if we have not yet established websocket connection
-     * and do not know if user has write access to a file.
-     * @deprecated - use readOnlyInfo
-     */
-    public get readonly() {
```

Comment:
I believe you will run into assert 0x124 here - you can't remove it before we eliminate usage in container runtime layer and wait for 2 release to pass (due to back compat testing in our UTs).